README.md

```diff
@@ -12,10 +12,13 @@ jobs:
     name: Terraform
     uses: nrkno/github-workflow-terraform-config/.github/workflows/workflow.yaml@main
     with:
+      # Whether to enable the Terraform checks. Used for testing this workflow itself.
```

Comment:
should remove the "Used for testing this workflow itself", as this has other legitimate use cases